Demo/Application/Features/PayPalWebCheckoutViewController.swift

```diff
@@ -14,23 +21,27 @@ class PayPalWebCheckoutViewController: PaymentButtonBaseViewController {
         let payPalCheckoutButton = createButton(title: "PayPal Checkout", action: #selector(tappedPayPalCheckout))
         let payPalVaultButton = createButton(title: "PayPal Vault", action: #selector(tappedPayPalVault))
         let payPalPayLaterButton = createButton(title: "PayPal with Pay Later Offered", action: #selector(tappedPayPalPayLater))
-        let payPalAppSwitchButton = createButton(title: "PayPal App Switch Flow", action: #selector(tappedPayPalAppSwitchFlow))
+        let payPalAppSwitchButton = createButton(title: "PayPal App Switch", action: #selector(tappedPayPalAppSwitchFlow))
```

Comment:
Do we also want to update the method to `tappedPayPalAppSwitch` since we removed `flow` from the title?

Reply:
Good eye, yes! Done in 7d54d3fdf7b76c397a2aab022387d38187fbada9